### PR TITLE
Fix eslint errors (tests)

### PR DESCRIPTION
**Pull request by herenow**
Our lint tests weren't passing.

This lines of code had the wrong indentation.

### CODE REVIEW

File: lib/fs_utils/write.js [hunk 1]
```diff
@@ -14,14 +14,16 @@ const changedSince = startTime => generated => {
...
   const helpers = sourceFileJoinConfig.pluginHelpers;
-  return Object.keys(sourceFileJoinConfig)
-  .filter(key /* generatedFilePath */ => {
-    if (key === 'pluginHelpers') return;
-    if (isHelper) return helpers.includes(key);
-
-    const checker = sourceFileJoinConfig[key];
-    return checker(path);
-  });
+  return (
```
Comment (shvaikalesh) [inline lib/fs_utils/write.js]: Why extra braces?

File: lib/watch.js [hunk 2]
```diff
@@ -330,8 +330,7 @@ class BrunchWatcher {
         logger.warn('Attempting to recover from failed NPM requires by running `npm install`...');
         this.restartBrunch('npm');
       }
-    })
-    .then(() => {
+    }).then(() => {
```
Comment (shvaikalesh) [inline lib/watch.js]: 👍

File: lib/plugins.js [hunk 3]
```diff
@@ -83,13 +83,15 @@ const loadPackages = rootPath => {
...
 
 const getPlugins = (packages, config) => {
-  return packages
-  .filter(plugin => plugin && plugin.prototype && plugin.prototype.brunchPlugin)
-  .map(Plugin => {
-    const instance = new Plugin(config);
-    instance.brunchPluginName = Plugin.brunchPluginName;
-    return instance;
-  });
+  return (
```
Comment (shvaikalesh) [inline lib/plugins.js]: I believe `.filter` and `.map` should be indented, without extra braces, they should fail `	
no-extra-parens` rule.

**Comment by herenow** [comment]
@shvaikalesh 👍  I've removed the extra braces

**Comment by shvaikalesh** [comment]
Awesome! @herenow, thanks for the contribution.

**Pull request merged**